strawberry_django_plus/relay.py

```diff
@@ -329,10 +329,14 @@ class Node(abc.ABC):
 
     All types that are relay ready should inherit from this interface and
     implement the following methods.
+    Attributes:
+        id_field:
```

Comment:
What do you think of calling it `id_attr`? To be explicit that it is the attribute to retrieve the id and not a possibly strawberry field.

Reply:
sounds good